includes/elements/section.php

```diff
@@ -288,6 +288,40 @@ protected function _register_controls() {
 					'extended' => __( 'Extended', 'elementor' ),
 					'wide' => __( 'Wide', 'elementor' ),
 					'wider' => __( 'Wider', 'elementor' ),
+					'custom' => __( 'Custom', 'elementor' ),
+				],
+			]
+		);
+
+		$this->add_responsive_control(
+			'gap_columns-custom',
+			[
+				'label' => __( 'Custom Columns Gap', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'range' => [
+					'px' => [
+						'min' => 0,
+						'max' => 500,
+					],
+					'%' => [
+						'min' => 0,
+						'max' => 100,
+					],
+					'vh' => [
+						'min' => 0,
+						'max' => 100,
+					],
+					'vw' => [
+						'min' => 0,
+						'max' => 100,
+					],
+				],
+				'size_units' => [ 'px', '%', 'vh', 'vw' ],
+				'selectors' => [
+					'{{WRAPPER}} .elementor-column-gap-custom > .elementor-column > .elementor-element-populated' => 'padding: {{SIZE}}{{UNIT}};',
```

Comment:
1. If there is an inner section, it will be affected by this rule.
2. You don't need the `.elementor-column-gap-custom` selector, since this selector is active only when `custom` is selected.

Reply:
1. No, it does not affect the inner section.

2. We must use this class, when we remove it, the default gap is applied